ClientApp/viking-vault/src/UserDashboard/TransactionList.tsx

```diff
@@ -75,22 +81,40 @@ class TransactionList extends React.Component<any, IState> {
     }
 
     private getImageForTransactionType(transactionType: string) {
-        if(transactionType.toLowerCase() === "transfer") {
+        if(transactionType.toLowerCase() === transactionTypeEnum.transfer) {
             return userIcon;
         }
-        if(transactionType.toLowerCase() === "exchange") {
+        if(transactionType.toLowerCase() === transactionTypeEnum.exchange) {
             return exchange;
         }
         return shoppingCart;
     }
 
+    private formatDetails(tranasction: ITransaction) {
+        if(tranasction.type.toLowerCase() === transactionTypeEnum.exchange) {
+            return "Exchange";
+        }
+        return tranasction.details;
+    }
+
+    private formatAmount = (transation: ITransaction) => {
+        if(transation.type === "payment") {
+            return -transation.amount;
+        }
+        if(transation.amount > 0) {
+            return `+${transation.amount}`;
+        }
+        return transation.amount;
```

Comment:
Seeing a lot of return statements is confusing, can you get rid of them and only use them once?

Reply:
There are like 3 of them but fine :)). Done